Makefile

```diff
@@ -296,6 +297,17 @@ build-ova-provider-server-image: check_container_runtime
 		$(BAZEL_OPTS) \
 		--action_env CONTAINER_CMD=$(CONTAINER_CMD)
 
+build-image-converter-image: check_container_runtime
```

Comment:
the changes in forklift-controller look good overall but adding a new image for qemu-img that already exists in the virt-v2v image is debatable, did you try to see if we can reuse the el9 image with virt-v2v instead?

Reply:
maybe we don't need to binary part if all it does is to invoke qemu-img, we can enhance [the entrypoint script in that image](https://github.com/kubev2v/forklift/blob/main/virt-v2v/cold/entrypoint) instead

Reply:
I thought about it, but I feel like the progress monitoring part will be simpler if it's wrapped in go 😅

Reply:
:)
we anyway need to sort out progress reporting with the populator controller for virt-v2v (it's really about time to start working on that!)